libmscore/layout.cpp

```diff
@@ -4173,6 +4173,10 @@ void Score::layoutSystemElements(System* system, LayoutContext& lc)
 
 //---------------------------------------------------------
 //   collectPage
+//    append systems to the page until you find a page break
+//    or run out of Movements (MasterScores)
+//
+//    called after LayoutContext::getNextPage()
```

Comment:
Good to see this kind of comments! Makes it more clear.

Reply:
These comments are so good that you should prefix them with `///` (three slashes) to expose them to Doxygen. Don't use three slashes before the function name (`//    collectPage`) or lines with dashes (`//-----------`).